src/Giveaway.js

```diff
@@ -915,17 +915,21 @@ class Giveaway extends EventEmitter {
                 }
                 resolve(winners);
             } else {
-                const embed = this.fillInEmbed(options.messages.error.embed);
-                channel.send({
-                    content: this.fillInString(options.messages.error.content || options.messages.error),
-                    embeds: embed ? [embed] : null,
-                    allowedMentions: this.allowedMentions,
-                    reply: {
-                        messageReference:
-                            typeof options.messages.error.replyToGiveaway === 'boolean' ? this.messageId : undefined,
-                        failIfNotExists: false
-                    }
-                });
+                if (options.messages.replyWhenNoWinner) {
```

Comment:
```suggestion
                if (options.messages.replyWhenNoWinner !== false) {
```
i think it's better to only deactivate with a real boolean